mne/epochs.py

```diff
@@ -829,8 +835,20 @@ def _compute_mean_or_stderr(self, picks, mode='ave'):
     def _evoked_from_epoch_data(self, data, info, picks, n_events, kind):
         """Create an evoked object from epoch data."""
         info = deepcopy(info)
+        if self.name not in ['Unknown', None]:
+            comment = self.name
+        else:
+            if len(self.event_id.keys()) == 1:
+                comment = self.event_id.keys()[0]
```

Comment:
this is a generator on Py3k so this is an error (need `list(self.event_id.keys())[0]`)